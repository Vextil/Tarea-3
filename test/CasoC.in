CrearLista 10 Migliore Sandoval Valdez MacEachen Rodriguez MarcelNovick Aguiar Japo Urreta T8ny Zalayeta FIN
ImprimirLista 10
RemoverDeLista 10 2
ImprimirLista 10
Apilar Guruceaga 1
Apilar Macaluso 13
Apilar Leyes 33
Apilar JM19 19
Desapilar
Desapilar
Desapilar
Desapilar
Desapilar
CrearLista 9 Munua Romero Gorga DeLosSantos Diaz Arismendi Porras Barcia Ramos DePena Alonso FIN
ConstruirSet 10 1
ConstruirSet 9 2
PerteneceASet 1 9
AgregarProcesos a 15 b 23 c 88 d 19 e 10 f 57 FIN
ImprimirProcesos
Fin