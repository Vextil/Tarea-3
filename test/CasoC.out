1>OK
2>NO ordenada. Migliore Sandoval Valdez MacEachen Rodriguez MarcelNovick Aguiar Japo Urreta T8ny Zalayeta 
3>Sandoval
4>NO ordenada. Migliore Valdez MacEachen Rodriguez MarcelNovick Aguiar Japo Urreta T8ny Zalayeta 
5>OK
6>OK
7>OK
8>OK
9>(19,JM19)
10>(33,Leyes)
11>(13,Macaluso)
12>(1,Guruceaga)
13>La pila estaba vacia. 
14>OK
15>OK
16>OK
17>NO NO NO NO NO NO NO NO NO NO NO 
18>Hay 7 procesos.
19>
(57,f) 
(10,e) 
(23,b) (19,d) 
(15,a) (88,c) 
(100,boot) 
20>Fin
